### ISSUE TITLE
Blog: Gradient descent in spiking neural networks for low-power inference

### ISSUE THREAD

**Issue by ikedaosushi** [ikedaosushi/tech-news#21415]
Blog: Gradient descent in spiking neural networks for low-power inference<br>
<br>
https://ift.tt/38waU8k
